### PR TITLE
Fix interface and notify

### PR DESCRIPTION
**Pull request by PhuocBao**

### CODE REVIEW

File: app/src/main/java/com/framgia/music/service/SongService.java [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+import android.content.IntentFilter;
 import android.content.ServiceConnection;
 import android.media.MediaPlayer;
 import android.net.Uri;
 import android.os.Binder;
 import android.os.Build;
 import android.os.IBinder;
 import android.support.annotation.Nullable;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.content.ContextCompat;
```
Comment (le-quang-dao) [inline app/src/main/java/com/framgia/music/service/SongService.java]: ERROR: Unused import: android.support.v4.content.ContextCompat. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

File: app/src/main/java/com/framgia/music/service/SongService.java [hunk 2]
```diff
@@ -87,36 +94,61 @@ public void unbindService(ServiceConnection conn) {
...
+                        .setAutoCancel(true)
+                        .setCustomBigContentView(remoteViews);
+        if (manager != null) {
+            manager.notify(NOTIFY_ID, builder.build());
         }
+        startForeground(NOTIFY_ID, builder.build());
+    }
+
+    private int getNotificationIcon() {
+        boolean useWhiteIcon = (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP);
```
Comment (le-quang-dao) [inline app/src/main/java/com/framgia/music/service/SongService.java]: ERROR: Line is longer than 100 characters (found 109).

File: app/src/main/java/com/framgia/music/service/SongService.java (hunk 1 above)
Comment (le-quang-dao) [inline app/src/main/java/com/framgia/music/service/SongService.java]: WARNING: [UnusedImports] Avoid unused imports such as 'android.support.v4.content.ContextCompat'
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#UnusedImports

File: app/src/main/java/com/framgia/music/service/DownloadService.java [hunk 3]
```diff
@@ -117,6 +161,51 @@ public void resumeMusic() {
...
+        remoteViewsDownload.setOnClickPendingIntent(R.id.image_previous, pendingPrevious);
+        if (mMediaPlayer != null && mMediaPlayer.isPlaying()) {
+            remoteViewsDownload.setImageViewResource(R.id.image_play, R.drawable.ic_pause);
+        } else if (mMediaPlayer != null) {
+            remoteViewsDownload.setImageViewResource(R.id.image_play, R.drawable.ic_play);
+        }
+        NotificationManager manager =
+                (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        NotificationCompat.Builder builder =
+                new NotificationCompat.Builder(this).setSmallIcon(R.drawable.ic_headphone)
```
Comment (le-quang-dao) [inline app/src/main/java/com/framgia/music/service/DownloadService.java]: WARNING: [IconColors] Notification icons must be entirely white
 Notification icons and Action Bar icons should only white and shades of gray. See the Android Design Guide for more details. Note that the way Lint decides whether an icon is an action bar icon or a notification icon is based on the filename prefix: `ic_menu_` for action bar icons, `ic_stat_` for notification icons etc. These correspond to the naming conventions documented in http://developer.android.com/guide/practices/ui_guidelines/icon_design.html

**Comment by daolq3012** [comment]
OK!

**Pull request merged**